internal/nodejs-action/action.yml

```diff
@@ -22,6 +22,15 @@ inputs:
     type: string
     required: false
 
+outputs:
+  attestation-name:
```

Comment:
should this be called provenance-name or attestation-name?

Reply:
I think `attestation-name` is more accurate since it's the attestation generated by `generate-attestations` and signed by `sign-attestations`.

This is only really used internally by the workflow so it shouldn't be exposed to users anyway.